LanguageFeatures/Enhanced-Enum/grammar_A10_t04.dart

```diff
@@ -9,16 +9,15 @@
 /// for the class, this restriction ensures that there is no conflict with that
 /// declaration.
 ///
-/// @description Check that it's a compile-time error if the enum declaration
-/// contains a static setter with the name `values`, and its argument is not a
-/// supertype of the generated member `values`.
+/// @description Check that it's not an error if the enum declaration contains a
+/// static setter with the name `values`, and its argument is not a supertype of
```

Comment:
Let's just keep the naming like in the specification:
```suggestion
/// static setter with the name `values=`, and its argument is not a supertype of
```

Reply:
Updated.